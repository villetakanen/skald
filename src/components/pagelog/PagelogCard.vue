<template>
  <v-card class="sidebar"></v-card>
</template>
