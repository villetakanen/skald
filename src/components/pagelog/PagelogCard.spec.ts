import { shallowMount } from '@vue/test-utils'
import PagelogCard from './PagelogCard.vue'

describe('Mounted App', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(PagelogCard, {
      mocks: {
        $t: () => {}
      },
      propsData: {
        siteid: 'skald'
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
