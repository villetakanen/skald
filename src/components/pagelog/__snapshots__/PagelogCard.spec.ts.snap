// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mounted App renders correctly 1`] = `
<v-card-stub
  class="sidebar"
  loaderheight="4"
  siteid="skald"
  tag="div"
/>
`;
